azext_iot/_help.py

```diff
@@ -96,6 +96,39 @@
         az iot hub monitor-feedback -n {iothub_name} -d {device_id} -w {message_id}
 """
 
+helps[
+    "iot hub connection-string"
+] = """
+    type: group
+    short-summary: Manage IoT hub\'s connection string.
+"""
+
+helps[
+    "iot hub connection-string show"
+] = """
+    type: command
+    short-summary: Show the connection strings for an IoT hub.
+    examples:
+    - name: Show the connection strings for all IoT hubs in a subscription.
```

Comment:
We should append ' using the default policy and primary key.' (like in below examples)  For the by subscription/by resource group examples when --all is not provided.

Reply:
Fixed